eZ/Publish/Core/Persistence/Legacy/Content/Location/Gateway/DoctrineDatabase.php

```diff
@@ -401,12 +403,36 @@ public function moveSubtreeNodes(array $sourceNodeData, array $destinationNodeDa
         }
     }
 
-    private function isHiddenByParent($pathString, array $rows)
+    private function getHiddenNodeIds(int $contentObjectId)
```

Comment:
side note: moving method position in the class, while convenient during dev, at the same time it obfuscates the diff here.

Reply:
yes, so you prefer pretty code or pretty diffs? Again, I used the same order in ezplatform-kernel, in order to easy merging in the future....

Reply:
> yes, so you prefer pretty code or pretty diffs? Again, I used the same order as in ezplatform-kernel, in order to easy merging in the future....

Ok, I didn't know it's a copy-paste. Let's keep it for now, no need to spend extra time. Usually I try to keep diffs as small as possible so reviewing them raises less questions and is faster. But sometimes I go extra mile too if I can't stand the code I'm working with :)